pkg/restore/restore.go

```diff
@@ -834,13 +839,12 @@ func (ctx *restoreContext) shouldRestore(name string, pvClient client.Dynamic) (
 
 // crdAvailable waits for a CRD to be available for use before letting the
 // restore continue.
-func (ctx *restoreContext) crdAvailable(name string, crdClient client.Dynamic) (bool, error) {
+func (ctx *restoreContext) crdAvailable(name string, crdClient client.Dynamic, resourceTimeout time.Duration) (bool, error) {
```

Comment:
```suggestion
func (ctx *restoreContext) crdAvailable(name string, crdClient client.Dynamic) (bool, error) {
```
restoreContext already have resourceTimeout within its structs. Why is there a need to pass this function parameter? restoreContext should've already been set with this value.